torch/onnx/experimental/fonnx/v1/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .registry import ONNXRegistry
```

Comment:
need to include `registry.py` in v1 too due to dependencies.

Reply:
@abock maybe we should defer the optional registry as an option until we are interested in reviewing `ONNXRegistry`?

This class is probably too complex for a v1. It will be an optional argument anyway, so it won't affect BC for our beta tester users

Reply:
removed registry from v1